ci/docker/codecov/deploy.Dockerfile

```
@@ -0,0 +1,102 @@
+ARG BUILD_IMAGE
+
+# This is the folder where the project is built
+ARG BUILD=/DLA-Interface-build
+
+# This is where we copy the sources to
+ARG SOURCE=/DLA-Interface
+
+# Where "make install" should go / we copy the bare minimum
+# of binaries to here
+ARG DEPLOY=/root/DLA-Interface.bundle
+
+FROM $BUILD_IMAGE as builder
+
+ARG BUILD
+ARG SOURCE
+ARG DEPLOY
+
+# Build DLA-Interface
+COPY . ${SOURCE}
+
+SHELL ["/bin/bash", "-c"]
+
+# Build the project with coverage symbols
+RUN mkdir ${BUILD} && cd ${BUILD} && \
+    CC=${MPICH_PATH}/bin/mpicc CXX=${MPICH_PATH}/bin/mpicxx FC=${MPICH_PATH}/bin/mpif90 cmake ${SOURCE} \
+      -DCMAKE_BUILD_TYPE=Debug \
+      -DCMAKE_CXX_FLAGS="-O0 -Werror -fprofile-arcs -ftest-coverage" \
+      -DCMAKE_EXE_LINKER_FLAGS="-fprofile-arcs -ftest-coverage" \
+      -DDLAI_CI_RUNNER_USES_MPIRUN=ON \
+      -DDLAI_WITH_MKL=0 \
+      -DLAPACK_CUSTOM_TYPE=Custom \
+      -DLAPACK_CUSTOM_INCLUDE_DIR=${OPENBLAS_PATH}/include \
+      -DLAPACK_CUSTOM_LIBRARY="-L${OPENBLAS_PATH}/lib;openblas" \
```

In the last commits of the CMake refactoring I changed this.

```suggestion
      -DLAPACK_LIBRARY="-L${OPENBLAS_PATH}/lib;openblas" \
```